build.yaml

```diff
@@ -102,7 +102,24 @@ release:
                   find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \; 
                   rm -rf build
                   PHP_TOKENS=($(echo ${PHP_VERSION} | tr "-" " "))
-                  sudo ${PACKAGE_UTILITY_REMOVE} ${PHP_TOKENS[0]}-common
+                  # Installing packaging and testing connection
+                  sudo ${PACKAGE_INSTALL} packages/${PHP_TOKENS[0]}*.${PACKAGE_TYPE}
+                  php <<EOF
+                  <?php
+                  \$cluster = Cassandra::cluster()->build();
+                  \$keyspace = 'system';
+                  try{
+                    \$session = \$cluster->connect(\$keyspace);
+                  } catch (Cassandra\\Exception\\RuntimeException \$e) {
+                    echo 'Caught exception: ',  \$e->getMessage(), "\n";
```

Comment:
You could do something like `echo "Success" . PHP_EOL"` here and then check for that output in the bash if statement below.  If you're extra paranoid you could check `$e->getMessage()` for the particular error you were expecting and only echo success in that case, but I think that's overkill as we've proven that we've successfully loaded the module by virtue of throwing/catching the `Cassandra\Exception\RuntimeException`.